Client/Frontend/Login Management/BreachAlertsManager.swift

```diff
@@ -30,50 +30,57 @@ final public class BreachAlertsManager {
     static let detailColor = UIColor(red: 0.59, green: 0.11, blue: 0.11, alpha: 1.00)
     static let monitorAboutUrl = URL(string: "https://monitor.firefox.com/about")
     var breaches = Set<BreachRecord>()
-    var breachAlertsClient: BreachAlertsClientProtocol
-
-    init(_ client: BreachAlertsClientProtocol = BreachAlertsClient()) {
-        self.breachAlertsClient = client
+    var client: BreachAlertsClientProtocol
+    var profile: Profile!
+    private lazy var cacheURL: URL = {
+        return URL(fileURLWithPath: (try? self.profile.files.getAndEnsureDirectory())!, isDirectory: true).appendingPathComponent("breaches.json")
+    }()
+    init(_ client: BreachAlertsClientProtocol = BreachAlertsClient(), profile: Profile) {
+        self.client = client
+        self.profile = profile
     }
 
     /// Loads breaches from Monitor endpoint using BreachAlertsClient.
     ///    - Parameters:
     ///         - completion: a completion handler for the processed breaches
     func loadBreaches(completion: @escaping (Maybe<Set<BreachRecord>>) -> Void) {
-        self.breachAlertsClient.fetchData(endpoint: .breachedAccounts) { maybeData in
-            guard let data = maybeData.successValue else {
-                completion(Maybe(failure: BreachAlertsError(description: "failed to load breaches data")))
+        if FileManager.default.fileExists(atPath: self.cacheURL.path) {
```

Comment:
`guard` instead of `if` pls

Reply:
This `if` is part of an `if`/`else` so is `guard still necessary? else: 
https://github.com/mozilla-mobile/firefox-ios/blob/209043929b3a3848228477d6d28c2ff265218b2b/Client/Frontend/Login%20Management/BreachAlertsManager.swift#L81-L85
Not sure if `guard` would make it more readable but open to suggestions of course